public/admins.html

```
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/adminsStyles.css"/>
+    <link rel="stylesheet" href="css/normalize.css" />
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap");
+    </style>
+    <title>MindSet</title>
+  </head>
+  <body>
+    <nav>
+      <img src="img/Logo-MindSet.png" alt="MindSet logo" />
+    </nav>
+    <section>
+      <div id="resource-table">
+        <h1>Admins</h1>
+        <table>
+          <thead>
+            <tr>
+              <th>Id</th>
+              <th>name</th>
+              <th>e-mail</th>
+              <th>username</th>
+              <th>password</th>
+              <th>superadmin</th>
+              <th>edit</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>1</td>
+              <td>3</td>
+              <td>3</td>
+              <td>4</td>
+              <td>5</td>
+              <td><button class="editButton"><img src="img/Icon-edit.png" alt="EditAdmin"/></button></td>
+            </tr>
+          </tbody>
+        </table>
+        <button id="back-button">Back></button>
+        <div id="background-modal">
+          <div id="create-modal">
+            <h2>Update Admin</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure
+              nulla alias nisi quidem doloribus.
+            </p>
+            <form>
+                <fieldset>
+                    <label for="name">Name</label>
+                    <input
+                        type="text"
+                        id="adminName"
+                        name="name"
+                        value=""
+                    />
+                    <span id="Error1" class="Error-msg">Error</span>
```

@JamirMansilla we could add an class to hide the error with display: none so the span takes up the space. So when we have an error, we remove this class and the text inserted won't push down the input that follows it.

---

i dont be sure if i'm following you but I think this looks good at the end and works fine. so i would like to keep it that way